src/test/frontend/db/query_dsl_test.cljs

```diff
@@ -2,30 +2,44 @@
   (:require [cljs.test :refer [are deftest testing use-fixtures is]]
             [clojure.string :as str]
             [frontend.db :as db]
-            [frontend.db.config :refer [test-db] :as config]
             [frontend.db.query-dsl :as query-dsl]
-            [frontend.handler.repo :as repo-handler]))
+            [frontend.test.helper :as test-helper :refer [load-test-files]]))
 
 ;; TODO: quickcheck
 ;; 1. generate query filters
 ;; 2. find illegal queries which can't be executed by datascript
 ;; 3. find filters combinations which might break the current query implementation
 
+(use-fixtures :each {:before test-helper/start-test-db!
+                     :after test-helper/destroy-test-db!})
+
 ;; Test helpers
 ;; ============
-(defn- load-test-files [files]
-  (repo-handler/parse-files-and-load-to-db! test-db files {:re-render? false}))
+
+(def dsl-query*
+  "When $EXAMPLE set, prints query result of build query. Useful for
+   documenting examples and debugging"
+  (if (some? js/process.env.EXAMPLE)
```

Comment:
This allows us to generate some example rule usage for users - https://gist.github.com/logseq-cldwalker/796202d55897fd34ea3d8c3bb3401ae0

Reply:
Very cool!